SkillChat.Client.ViewModel/Interfaces/IColorGenerator.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SkillChat.Client.ViewModel.Interfaces
+{
+    public interface IColorGenerator
+    {
+        object GetColor(string colorName);
```

Comment:
Этот метод должен получать не название цвета, а UserId, и всю работу по выбору цвета и его прикреплению к пользователю пусть делает реализация этого метода.
```suggestion
        object GetColor(string userId);
```